KVMarketSquare/Scenes/SearchView.swift

```diff
@@ -152,26 +152,20 @@ struct SellerResultsListView: View {
         switch task.result {
         case .success(let model):
             List(model.data) { store in
-                Button {
-                    let sellerStore = SellerAppData(
+                NavigationLink {
+                    StoreWebView(store: SellerAppData(
```

Comment:
i am going to guess this has something to do with the way this view immediately tries to load the task on init - we can move that into the .none of the body result case in this view

Reply:
i will do that in a pr in a minute, hold on